cluster/upgrade_tool.sh

```diff
@@ -0,0 +1,19 @@
+tool=$1
```

Comment:
Recommend starting with this preamble given how many things could potentially fail in this script:

```
#!/usr/bin/env bash

set -e
```
That way it will die straight away if say the `wget` fails.